python/cugraph/jaccard/jaccard_wrapper.pyx

```diff
@@ -79,46 +46,20 @@ cpdef nvJaccard(input_graph):
     >>> G.add_edge_list(sources,destinations,None)
     >>> jaccard_weights = cuGraph.jaccard(G)
     """
-
     cdef uintptr_t graph = input_graph.graph_ptr
     cdef gdf_graph* g = <gdf_graph*>graph
 
-    cdef uintptr_t adjList_ptr = <uintptr_t>g.adjList
-    if adjList_ptr is 0:
-        err = gdf_add_adj_list(<gdf_graph*>graph)
-        cudf.bindings.cudf_cpp.check_gdf_error(err)
-
-    cdef uintptr_t offsets_ptr = <uintptr_t>g.adjList.offsets.data
-    cdef uintptr_t indices_ptr = <uintptr_t>g.adjList.indices.data
-    cdef uintptr_t edge_value_ptr = <uintptr_t>g.adjList.edge_data
-    cdef uintptr_t value_ptr
-    val_type = gdf_to_cudaDataType[libgdf.GDF_FLOAT32].value
-    if edge_value_ptr:
-        value_ptr = <uintptr_t>g.adjList.edge_data.data
-        val_type = gdf_to_cudaDataType[g.adjList.edge_data.dtype].value
+    if g.adjList:
+        e = g.adjList.indices.size
     else:
```

Comment:
I there is no adj list, it should be created (like in the previous version) by calling `gdf_add_adj_list`

Reply:
That is taken care in c++ side in cugraph.cu line 393

Reply:
Should we still do it at the python level so that the bindings don't make any assumption on the behavior of the C++ part?
I wonder if doing this at C++ vs Python level could be related to issue #167

Reply:
This is not blocking the PR I am just curious

Reply:
can look into it

Reply:
Thanks! I approved this PR, will merge it after getting James confirmation. Let's take care of this as part of the PR for #167.